app/models/parent_object.rb

```diff
@@ -139,4 +139,28 @@ def iiif_manifest
     return @iiif_manifest if @iiif_manifest
     @iiif_manifest = @iiif_presentation.manifest if iiif_presentation.valid?
   end
+
+  def child_captions
+    captions = []
```

Comment:
This implementation is correct, but not optimal. Let’s spend a couple minutes this morning tidying this up

Reply:
This has been refactored